src/server/checkUserInfo.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+const cryptoJs = require('./crypto.js');
+let bodyParser = require('body-parser');
+const express = require('express');
+const url = 'mongodb://localhost:27017/epam';// already defined;
+const app = express();
+let mongodb = require('mongodb');
+let MongoClient = mongodb.MongoClient;
+app.use(bodyParser.json()); // for parsing application/json
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.static('dist'));
+app.get('/heartbeat', (req, res) => {
```

Comment:
you dont need to re-declare these things, they are already declared in the server definition